fvm/src/syscalls/actor.rs

```diff
@@ -112,3 +113,11 @@ pub fn install_actor(
     let typ = context.memory.read_cid(typ_off)?;
     context.kernel.install_actor(typ)
 }
+
+pub fn actor_balance(
+    _context: Context<'_, impl Kernel>,
+    _addr_off: u32,
+    _addr_len: u32,
+) -> Result<sys::TokenAmount> {
+    todo!()
+}
```

Comment:
`balance_of` sounds like a better name to prevent the stutter (`actor::actor_balance`).

This seems relatively easy to implement.

Reply:
ok, will rename.

Reply:
done and implemnted.